server/autotest_server/__init__.py

```diff
@@ -97,12 +104,16 @@ def _create_test_script_command(tester_type: str) -> str:
     Return string representing a command line command to
     run tests.
     """
-    import_line = f"from testers.{tester_type}.{tester_type}_tester import {tester_type.capitalize()}Tester as Tester"
+    resource_settings = get_resource_settings(config)
+    import_line = (
```

Comment:
revert this change

Reply:
This formatting change seems to come from black.

Reply:
It's not just a line break change, there are now two consecutive strings here. Please revert this.

Reply:
Sorry, completely missed that. Reverted.